packages/workflow/src/WorkflowDataProxy.ts

```diff
@@ -276,20 +276,23 @@ export class WorkflowDataProxy {
 			}
 
 			if (!that.workflow.getNode(nodeName)) {
-				throw new ExpressionError(`"${nodeName}" node doesn't exist`, {
+				throw new ExpressionError("Referenced node doesn't exist", {
 					runIndex: that.runIndex,
 					itemIndex: that.itemIndex,
+					nodeCause: nodeName,
+					description: `The node <strong>'${nodeName}'</strong> doesn't exist, but it's used in an expression here.`,
```

Comment:
Should add a description key of some sorts (perhaps `type`) here instead, and keep the actual description in editor, so that we can use localization?

Reply:
Should it be both (a description and a key/type)?
- In the backend and API the non-localized description is used
- The frontend uses `type` to get a message/description from the i18n dictionary (and can fallback to the not-localized version if the i18n key doesn't exist)

Or maybe the not-localized description is not needed? lmk